docs/Tutorials/Advanced/ticketing/drop.md

```diff
@@ -0,0 +1,390 @@
+- `methodName`: The function to be called on the receiver contract.  
+- `args`: A stringified JSON object of all the arguments to be passsed into `methodName`.  
+- `attachedDeposit`: The yoctoNear deposit attached to the function call when the key is used.  
+
+### Tailoring to Ticketing Requirements
+
+To ensure each key has two key uses with a passwort protected first use, the following `config` can be added to `createDrop`.
+
+```js
+// 2 Uses per key
+config: {
+    usesPerKey: 2
+},
+
+// Create base password and ensure only first key use is password protected
+basePassword: "event-password",
+passwordProtectedUses: [1],
+```
+
+:::info
+As disucssed in the [introduction](drop.md#introduction), the first key use should not call anything, and the second should call `nft_mint`. To do this, a value of `null` can be passed in to the first index of the `methods` array
+:::
+
+On top of the required `receiverId`, `methodName`, `args`, and `attachedDeposit` parameters, a few others will be used used in `methods` to better utilize the SDK. These parameters tell Keypom where to inject arguments. 
+
+- `accountIdField`: Specifies what field Keypom should auto-inject the claiming account's `accountId`into when calling the function.  
+- `dropIdField`: Specifies what field Keypom should auto-inject the drop's `dropId` into when calling the function.  
+
+These parameters will be used alongisde the [`createNFTSeries`](../../../keypom-sdk/modules.md#createnftseries) SDK method.  
+
+In this case, the drop will be interfacing with an NFT contract deployed at `nft-v2.keypom.testnet`. This is a contract specifically made to work with the SDK to allow you to seamlessly create NFTs and NFT Series to attach to your drops. 
+
+:::note
+If you wish to use a different NFT contract for your POAP, ensure you know the contract's interface and tailor the `methods` arguments accordingly.
+:::
+
+```js reference
+https://github.com/keypom/keypom-js/blob/5e4b4744a16c727d96d235282020c186edd0b0b5/docs-advanced-tutorials/ticket-app/frontend/utils/createTickDrop.js#L30-L61
+```
+
+:::note
+Calling `claim` here before creating the NFT Series will fail. This only applies if you choose to include a POAP using `nft_mint` in the second `claim`. 
+:::
+
+ 
+
+You may have noticed that the syntax for the `nft_mint` function call looks slightly different from what was used in the [FC drop tutorial](../../Basics/fc-drops.md#creating-drop-with-function-call-data). This is because an NFT series is being minted. The NFT series must be defined before `claim` can be called and will require the drop ID, as you'll see in the next section. 
+
+:::info
+An NFT series can be thought of as a bucket of NFT token IDs that *all* share similar information. This information comes in the form of metadata, royalties, price etc. For more on NFT series, see NEAR's [NFT tutorial](https://docs.near.org/tutorials/nfts/series#nft-collections-and-series)
```

Comment:
Make this not an info tag and put it maybe at the top of the making NFT series section. Let's also reword a bit:

An NFT series is a set of NFTs that share the same artwork, title, description etc. The only thing that differs between tokens is their unique ID. For a full tutorial about the series contract, see NEAR's ....